Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+## Coaching Guide
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
+
+## How do I know if I am coaching?
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
+
+It is important that you do not vote for a goal if you are coaching so check with the coach team lead each week you are level 3 or above to determine if you are on coach-duty or not.
+
+## What does coaching entail?
+
+Check out the [Technical Coach](../Game_Manual/Levels_and_Roles.md#technical-coach) outline in [Levels and Roles](../Game_Manual/Levels_and_Roles.md).
+
+### Okay, but what does my day actually look like?
+
+As a coach you have 3 main responsibilities:
+
+1. **Maintain available hours for coach-queue appointments**. These are half-hour blocks learners can schedule through the coach-queue app when they are feeling blocked. Your google calendar will be the source of your availability so make sure you block off time you do not have available for appointments there.
+1. **Complete code reviews from the Prrr app**. Make yourself available to answer questions learners may have regarding your review of their code/project.
+1. **Escalating code blocks and issues to SEPs if you are unable to resolve the learners block during your appointment** (ideally, if after 15 minutes you do not feel you are on a solid path to debugging the issue, a coach should find an available SEP to bring in.)
+
+**Tip**: It is a good practice to block off time at the end and beginning of each day to focus on code reviews, and to keep as much middle-of-the-day time as possible available for coach-queue appointments.
+
+## What relevant skills does coaching teach me?
```

Comment:
This whole section is weird. A lot of odd questions. Maybe we can re-word this to be in the tips section?